### PR TITLE
Exit with proper status code when specs dir is empty or doesn't exists.

### LINKED ISSUE #1405: Gauge Returns Exit Code 0 When It Fails To Run
### Expected behavior
1. When attempting to run tests ("Gauge run...") in a folder that does not contain a Gauge project file, the Exit Code should be non-zero to denote that the run did not complete successfully.
2. When attempting to run tests in a folder that contains a Gauge project file but you do not point Gauge to a valid spec file or folder containing specs (e.g. "Gauge run adsfasfa" where there's no folder adsfasfa), the Exit Code should be non-zero to denote that the run did not complete successfully.

### Actual behavior
Exit Code 0 is returned.

### Steps to reproduce
1. Open up a command or Powershell window (only tested on Windows) to a directory that does not contain a Gauge project file.
2. Run "Gauge run specs"
3. Check the exit code using either "echo %errorlevel%" (CMD) or "$LASTEXITCODE" (Powershell)
---
1. Open up a command or Powershell window (only tested on Windows) to a directory that does contain a Gauge project file.
2. Run "Gauge run <random name>" where <random name> is a string that does not correspond to a folder in the current directory.
3. Check the exit code using either "echo %errorlevel%" (CMD) or "$LASTEXITCODE" (Powershell)

### Gauge version
Gauge version: 1.0.3
Commit Hash: ff6c0c3

Plugins
-------
csharp (0.10.5)
dotnet (0.1.1)
html-report (4.0.6)
screenshot (0.0.1)

### PR DESCRIPTION
**Pull request by negiDharmendra**
Fixes #1405

### CODE REVIEW

File: util/fileUtils.go [hunk 1]
```diff
@@ -159,12 +160,23 @@ func IsDir(path string) bool {
 
 // GetSpecFiles returns the list of spec files present at the given path.
 // If the path itself represents a spec file, it returns the same.
+var exitWithMessage = func(message string) {
+	logger.Errorf(true, message)
+	os.Exit(1)
+}
```
Comment (BugDiver) [inline util/fileUtils.go]: Can we use `logger.Fatal` here?

File: util/fileUtils.go [hunk 2]
```diff
@@ -159,12 +160,23 @@ func IsDir(path string) bool {
 
 // GetSpecFiles returns the list of spec files present at the given path.
 // If the path itself represents a spec file, it returns the same.
+var exitWithMessage = func(message string) {
+	logger.Errorf(true, message)
+	os.Exit(1)
```
Comment (BugDiver) [inline util/fileUtils.go]: Can we use `logger.Fatal` here?

Comment (negiDharmendra) [reply util/fileUtils.go]: it prints some extra information 
```
Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://spectrum.chat/gauge

Your Environment Information -----------
```
I don't think we need this as the error message itself explains the issue.

Comment (BugDiver) [reply util/fileUtils.go]: Ok

**Pull request merged**